bin/download_submission.py

```diff
@@ -8,20 +8,36 @@
 def get_args():
     """Set up command-line interface and get arguments without any flags."""
     parser = argparse.ArgumentParser()
-    parser.add_argument("submission_id", type=str, help="The ID of submission")
+    parser.add_argument("--submission-id", "-s", type=str, required=True, help="The ID of submission")
+    parser.add_argument("--file-type", "-f", type=str, required=True, help="The type of file the submission should be")
 
     return parser.parse_args()
 
 if __name__ == "__main__":
     args = get_args()
     submission_id = args.submission_id
+    file_type = args.file_type
 
     syn = synapseclient.Synapse()
     syn.login(silent=True)
 
     submission = syn.getSubmission(submission_id, downloadLocation=".")
     entity_type = submission["entity"].concreteType
-    if entity_type != "org.sagebionetworks.repo.model.FileEntity":
-        open("dummy.txt", 'w').close()
+    file_path = submission["filePath"]
+    
+    # TODO: Eventually we want to abstract this logic into the `make_invalid_file` function
+    # in model-to-data's `run_docker.py`, and move that out somewhere else.
+    invalid_file = f"INVALID_predictions.{file_type}"
+    error_msg = None
 
-    print(entity_type)
+    if entity_type != "org.sagebionetworks.repo.model.FileEntity":
+        error_msg = (
```

Comment:
NIT: Sorry for not bringing this up before, but this could use `concrete_types.FILE_ENTITY` from the python client instead of writing out the concrete type string.

```
from synapseclient.core.constants import concrete_types
...
if entity_type != concrete_types.FILE_ENTITY:
...
```

Also, I think in the imports on this script `sys` is no longer needed.

Reply:
Updated. Looks good!

```
(synapseclient) w241:nf-synapse-challenge jmedina$ python bin/download_submission.py -s 9751432 -f 'csv'

UPGRADE AVAILABLE

A more recent version of the Synapse Client (4.7.0) is available. Your version (4.4.0) can be upgraded by typing:
    pip install --upgrade synapseclient

Python Synapse Client version 4.7.0 release notes

https://python-docs.synapse.org/news/

Copying existing file from /Users/jmedina/.synapseCache/4/154107004/test-olfactory-submission.csv to /Users/jmedina/Documents/forks/nf-synapse-challenge/test-olfactory-submission.csv
(synapseclient) w241:nf-synapse-challenge jmedina$ 
```